apps/nextjs-pages-router/src/pages/api/makeswift/[...makeswift].ts

```diff
@@ -2,6 +2,8 @@ import { MAKESWIFT_SITE_API_KEY } from '@/makeswift/env'
 import { runtime } from '@/makeswift/runtime'
 import { MakeswiftApiHandler } from '@makeswift/runtime/next/server'
 
+import '@/makeswift/components'
```

Comment:
Not sure if this one is needed. If it is, we need to update our docs as well: https://docs.makeswift.com/developer/pages-router/installation

Reply:
It's needed if you use the Smartling integration.

Reply:
I can remove these for now since we don't use these apps for translation right now.

Reply:
Actually. I'm going to leave this for pages router. It's non trivial to figure out why translation is failing when this is not included, so I would prefer to keep it. Since app router doesn't support translations right now I'll leave it our of the `route.ts` file there.

Reply:
Can we add a comment that this needed for the Smartling integration?